gooddata-java/src/main/java/com/gooddata/sdk/service/GoodDataSettings.java

```diff
@@ -197,7 +216,7 @@ public void setPollSleepSeconds(final int pollSleep) {
      * @return user agent string
      */
     public String getUserAgent() {
-        return userAgent;
+        return StringUtils.isNotBlank(userAgent) ? String.format("%s %s", userAgent, getDefaultUserAgent()) : getDefaultUserAgent();
```

Comment:
I'm not sure about use cases, but could you consider to add another method which returns the `userAgent` property without default agent? And maybe add more javadoc what returns this particular method.